backend/bin/main/app/ehrenamtskarte/backend/verification/webservice/schema/CardMutationService.kt

```diff
@@ -0,0 +1,116 @@
+package app.ehrenamtskarte.backend.verification.webservice.schema
```

Comment:
Where does this files come from ? :D

Reply:
argh. unsure. I think my vscode kotlin plugin generates these files. will remove this again ^^